functions/src/index.ts

```diff
@@ -0,0 +1,231 @@
+import * as axios from 'axios';
+import * as base64 from 'js-base64'
+import * as functions from 'firebase-functions';
+import * as  admin from 'firebase-admin';
+// Will need to set up admin credentials for first time use
+admin.initializeApp({
+    credential: admin.credential.applicationDefault(),
+    databaseURL: "https://tuneiq.firebaseio.com"
+});
+
+// Types -- TODO: find a better place to put these
+export type RoundDetails = {
+    url: string
+    tracks: GameTrack[]
+}
+
+export type GameTrack = {
+    id: string,
+    title: string,
+    artists: string[],
+    isAnswer: boolean,
+}
```

Comment:
genre?

Reply:
nvm - genres are associated with games right?

Reply:
yes with the game, I can add it to the track if you can see value in that

Reply:
nah dont see immediate value, may add later if needed